earth_enterprise/src/fusion/portableglobe/SConscript

```diff
@@ -106,7 +106,7 @@ gedecode = env.executable('gedecode',
 
 gerewritedbroot = env.executable(
     'gerewritedbroot',
-    ['gerewritedbroot.cpp'],
+    ['gerewritedbroot.cpp', 'rewritedbroot.cpp'],
```

Comment:
Yes, rewritedbroot is a separate file to make it easier to test functions in it. Compiling gerewritedbroot.cpp together with the test driver fails because both of them have a `main` function.